### PR TITLE
Fixes #65: Boolean filters cannot be loaded from URL

### LINKED ISSUE #65: Boolean filter values handled as strings
Filter values of `true|false` fail internal validation, resulting in an error message of:

> The specified value is invalid for the given operator. Operator: exists. Value: false

### PR DESCRIPTION
**Pull request by aroc**
## What does this PR do? How does it affect users?

It fixes a bug where the `exists` filter could not be properly loaded from the URL. This was because the filter is being treated as a `string` type filter instead of `boolean`. This fixes that bug and also adds test coverage for it.

## How should this be tested?

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:

- [x] Does the code compile without warnings (check shell, console)?
- [x] Do all tests pass?
- [x] Does the UI, pixel by pixel, look exactly as expected (check various screen sizes, including mobile)?
- [x] If the feature makes requests from the browser, inspect them in the Web Inspector. Do they look as expected (parameters, headers, etc)?
- [x] If the feature sends data to Keen, is the data visible in the project if you run an extraction (include link to collection/query)?
- [x] If the feature saves data to a database, can you confirm the data is indeed created in the database?

## Related tickets?
#65

### CODE REVIEW

File: test/unit/utils/FilterUtilsSpec.js [hunk 1]
```diff
@@ -22,6 +23,16 @@ describe('utils/FilterUtils', function() {
         };
         assert.strictEqual(FilterUtils.getCoercionType(filter), "String");
       });
+
+      it('String type when property_value can be parsed into a date', function () {
```
Comment (sfalkoff) [inline test/unit/utils/FilterUtilsSpec.js]: Perhaps this test should be described as `'Boolean type when filter exists'` ? ... or something like that

**Comment by sfalkoff** [comment]
@aroc thanks for fixing this. +1 after comment

File: test/unit/utils/FilterUtilsSpec.js (hunk 1 above)
Comment (aroc) [inline test/unit/utils/FilterUtilsSpec.js]: Whoops, good eye! I copied the previous test and didn't update the name.

**Comment by adambraimbridge** [comment]
Any chance of a patch release for this? It missed the 2.2.0 release in Jan.
https://github.com/keen/explorer/compare/v2.2.0...master
